client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -106,23 +122,16 @@ const LoggedInForm = React.createClass( {
 		return this.props.jetpackConnectAuthorize && this.props.jetpackConnectAuthorize.isAuthorizing;
 	},
 
-	dismissNotice() {
-		debug( 'Notice dissmissed' );
-	},
-
 	renderNotices() {
-		const { authorizeSuccess, authorizeError } = this.props.jetpackConnectAuthorize;
-		if ( authorizeSuccess ) {
-			return <JetpackConnectNotices noticeType="authorizeSuccess" onDismissClick={ this.dismissNotice } />;
-		}
+		const { authorizeError } = this.props.jetpackConnectAuthorize;
 		if ( authorizeError ) {
-			return <JetpackConnectNotices noticeType="authorizeError" onDismissClick={ this.dismissNotice } />;
+			return <JetpackConnectNotices noticeType="authorizeError" />;
 		}
 		return null;
 	},
 
 	renderFormControls() {
-		const { queryObject, isAuthorizing, autoAuthorize, authorizeSuccess } = this.props.jetpackConnectAuthorize;
+		const { queryObject, isAuthorizing, autoAuthorize, authorizeSuccess, plansURL, siteReceived } = this.props.jetpackConnectAuthorize;
```

Comment:
oh damn, I forgot about the API doing that 

![image](https://cloud.githubusercontent.com/assets/1554855/14474805/1f81e8ca-00ff-11e6-8cf0-b13aadfd67e0.png)


I guess it's fine to have it there, I just wanted to avoid having jetpack as the one who sets this! :)